src/components/breadcrumbs/README.md

```diff
@@ -9,15 +9,54 @@
 {% endraw %}
 ```
 
-Call **breadcrumbs** like a normal function, set values for arguments `'breadcrumbs'` to alter its content.
+Call **breadcrumbs** like a normal function, set values for arguments `'title', 'url'` to alter its content.
```

Comment:
That line is pretty generic across all components, it was intended to be a pointer to the sections of a macro you might want to edit.